builtin/rebase.c

```diff
@@ -198,6 +210,10 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 		OPT_NEGBIT(0, "quiet", &options.flags,
 			N_("be quiet. implies --no-stat"),
 			REBASE_VERBOSE | REBASE_DIFFSTAT),
+		OPT_BOOL(0, "no-verify", &ok_to_skip_pre_rebase,
+			N_("allow pre-rebase hook to run")),
```

Comment:
The `--no-verify` option should go either into its own commit, or into the commit that implements handling for the `pre-rebase` hook.

I would favor the latter.